packages/components/inputs/search-select-input/src/search-select-input-open.visualroute.js

```diff
@@ -1,4 +1,4 @@
-import SearchSelectInput from './search-select-input';
+import { SearchSelectInput } from '@commercetools-frontend/ui-kit';
```

Comment:
Just wondering if we have some rule about _imports_. I ask it because it seems natural to me to rely on relative paths when the file is in fact part the the component (package) and the absolute path (module resolution) when referencing the component from the outside.

Reply:
I have the same sentiment. 
However, If we use relative paths and the file we are importing is in typescript, the visualroute/visualspec file cannot resolve the imported package.

Reply:
Ah! Ok, I see.
Thanks for the explanation.